src/cmd/rawdns/main.go

```diff
@@ -56,9 +69,18 @@ func main() {
 		switch config[domain].Type {
 		case "containers":
 			// TODO there must be a better way to pass "domain" along without an anonymous function AND copied variable
+			var tlsConfig *tls.Config
+			if config[domain].TLSCert != "" && config[domain].TLSKey != "" {
+				var err error
+				tlsConfig, err = loadTLSConfig(config[domain].TLSCACert, config[domain].TLSCert, config[domain].TLSKey, config[domain].TLSVerify)
+				if err != nil {
+					log.Fatalf("Unable to load tls config for %s: %s", domain, err)
```

Comment:
Can you update this to match the prose style of the other `Fatalf` calls here? ie, `error: unable to load tls config for %s: %s\n`